src/App/ProductSearch/ProductSearch.jsx

```diff
@@ -37,9 +37,6 @@ export default class ProductSearch extends Component {
         >
           View All
         </button>
-        <p style={{display: this.props.goodSearch ? 'none' : ''}}>
```

Comment:
I took this out because it was showing up when paths existed, it was quick fix for the demo.